.github/workflows/test.yml

```diff
@@ -9,51 +9,6 @@ on:
       - "*"
 
 jobs:
-  teleporter-integration-test:
```

Comment:
Can we rename this file to be a little more specific now that it only contains one type of test?

Reply:
Can we combine `e2e.yml` into here?

Reply:
Put everything in test.yml